Gemfile

```diff
@@ -18,7 +20,7 @@ group :development, :test do
   gem "sqlite3" # Remove this when you choose a production database
   gem "factory_girl_rails", "~> 4.7"
   gem "timecop"
-  gem "webmock", require: false
+  gem "webmock"
```

Comment:
do we need this change? it looks like webmock is required in the `spec/spec_helper.rb` which should do it? - unless I'm missing something different